src/main/resources/Dockerfile

```diff
@@ -31,6 +33,7 @@ ENV TARANTOOL_INSTANCES_FILE=$TARANTOOL_INSTANCES_FILE
 ENV TARANTOOL_CLUSTER_COOKIE=$TARANTOOL_CLUSTER_COOKIE
 COPY $CARTRIDGE_SRC_DIR $TARANTOOL_WORKDIR
 WORKDIR $TARANTOOL_WORKDIR
-RUN cartridge build
-CMD cartridge start --run-dir=$TARANTOOL_RUNDIR --data-dir=$TARANTOOL_DATADIR \
+
+RUN rm -rf .rocks && cartridge build --verbose
```

Comment:
Is volume mounting the reason for removing .rocks?

Reply:
If there is a .rock file, then a rocks collision may occur

Reply:
Until we have implemented .dockerignore support in testcontainers-java-tarantool, we need to remove .rocks folder, because they can appear after building on host system, for example when you debug cluster for some reason